src/handbook/development/releases/process.md

```diff
@@ -3,85 +3,161 @@ navTitle: Release Process
 ---
 # Release Process
 
-There are two processes, one for major/minor point releases. The second for patch
-releases.
+We do two types or releases:
 
-## Major/minor point releases
+ - Scheduled releases at a regular [monthly cadence](./planning.md#cadence). This
+   encompasses the entire set of product repositories.
+ - Fix releases of individual repositories as needed
+
+## Scheduled Releases
+
+We publish a new release of FlowForge [every four weeks](./planning.md#cadence). Each
+release has a Release Manager who oversees the process. This role rotates around
+the team to ensure everyone is able to do it and that we don't have any single
+points of failure in the process.
```

Comment:
```suggestion
We publish a new release of FlowForge in line with our [release plan](./planning.md#cadence). Each
release has a Release Manager who oversees the process. This role rotates around
the team to ensure everyone is able to do it and that we don't have any single
points of failure in the process.
```

(suggestion)

Removing the words "every four weeks" means any change to cadence section negates the need for a secondary update that could potentially be missed.